src/client/components/pages/RoomAdmin/RoomAdmin.tsx

```diff
@@ -138,7 +175,40 @@ const RoomAdmin: FunctionComponent = (): ReactElement => {
   useEffect((): void => {
     void loadRooms();
   }, [loadRooms]);
-
+  /**
+   * Populates building names for the building filter dropdown.
+   */
+  const buildingDropDown = buildingNameList.map((buildingName) => (
+    { value: buildingName, label: buildingName }
+  ));
```

Comment:
I think that this list of buildings should be sorted alphabetically. Currently, the buildings are alphabetically sorted within each campus, but I think that it would be nice to sort all of them for a better user experience.
<img width="405" alt="image" src="https://user-images.githubusercontent.com/29589689/207969699-84a72a95-9f58-4631-8bcb-ecf185ba2a7e.png">